tests/functional/cron/testReindex.js

```diff
@@ -213,60 +215,82 @@ describe('UtapiReindex', () => {
             });
         }
 
-        beforeEach(done => {
-            reindex._scheduleJob();
-            // Wait until the scripts have started and finished reindexing.
-            async.series([
-                next => waitUntilLockHasValue({ value: 'true' }, next),
-                next => waitUntilLockHasValue({ value: null }, next),
-            ], done);
-        });
-
-        it('should reindex metrics', done => {
-            async.parallel([
-                next => {
-                    const params = {
-                        resource: {
-                            type: 'buckets',
-                            buckets: [mock.values.BUCKET_NAME],
+        const bucketCounts = [1, 1001];
+
+        bucketCounts.forEach(count => {
+            describe(`bucket listing with a length of ${count}`, () => {
+                const bucket = `${mock.values.BUCKET_NAME}-${count - 1}`;
```

Comment:
why `count - 1` not `count`?

Reply:
Because the bucket creation listing is zero-indexed [here](https://github.com/scality/utapi/pull/352/files#diff-6f8c535ed582ceee20e2ec4e8216539dR47). Sounds like it would be more intuitive to actually set `count` in the test and `+ 1` in the bucket creation.